igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -5,38 +5,38 @@
+    "In this notebook, you will learn how to perform a time series InSAR analysis using the popular Small BAseline Subset (SBAS) technique. SBAS is well suited for monitoring slow, consistent deformation of the earth's surface such as slow landslides, subsidence, fault creep and volcanic inflation. In this tutorial we will focus on a volcanic inflation use case at [Mt. Edgecumbe, Alaska](https://en.wikipedia.org/wiki/Mount_Edgecumbe_(Alaska)).\n",
     "\n",
-    "1. Use the [ASF Search Python package](https://docs.asf.alaska.edu/asf_search/basics/) to:\n",
-    "   - Search ASF's catalog for Sentinel-1 SAR products covering the [Ridgecrest earthquake](https://earthquake.usgs.gov/storymap/index-ridgecrest.html)\n",
-    "   - Select a reference scene to generate a baseline stack\n",
-    "   - Select a [short baseline subset (SBAS)](https://docs.asf.alaska.edu/vertex/sbas/) of scene pairs for InSAR processing\n",
+    "In April 2022, a seismic swarm near Mt. Edgecumbe in southeast Alaska suggested there be could be renewed activity at the volcano. However, since the last sign of activity at Mt. Edgecumbe was ~800 years ago there was no monitoring equipment installed at the volcano. This made it difficult to determine if the volcano really was coming out of dormancy. Luckily, the team at the Alaska Volcano Observatory were experience InSAR users, and were able to prove via a time series analysis that volcanic inflation had begun at the site in 2018. Without this analysis, the activity state of Mt. Edgecumbe would have taken much longer to determine, which could have led to serious consequences if the volcano erupted before the neighboring city of Sitka, Alaska was prepared.\n",
     "\n",
+    "This notebook will show you how to  perform the time series InSAR analysis that AVO team used to reach their conclusions. It follows the methodology detailed int their analysis recent paper [(Grapenthin et. al., 2022)](https://doi.org/10.1029/2022GL099464), which utilizes On Demand InSAR products from the Alaska Satellite facility and the MintPy time series analysis software. \n",
     "\n",
-    "2. Use the [HyP3 Python SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) to:\n",
+    "In this notebook we'll:\n",
+    "\n",
+    "1. Use the [HyP3 Python SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) to:\n",
     "   - Request On Demand InSAR products from ASF HyP3\n",
     "   - Download the InSAR products when they are done processing\n",
-    "\n",
-    "\n",
-    "3. Use [GDAL](https://gdal.org/api/index.html#python-api) and [MintPy](https://mintpy.readthedocs.io/en/latest/) to:\n",
-    "   - Prepare the InSAR products for MintPy\n",
-    "   - perform a time-series analysis with MintPy\n",
-    "   \n",
+    "2. Use [MintPy](https://mintpy.readthedocs.io/en/latest/) and various Python utilities to:\n",
```

Comment:
Not sure how to format this properly in the suggestions (when I tried it did funny things), but I'd suggest adding a blank line before the second numbered item, just to give some visual space.

Reply:
will do!

Reply:
It looks like adding the space also changes the alignment of the numbers from 1 to 2. 😢